src/main/java/uk/gov/hmcts/reform/professionalapi/controller/LegacyPbaController.java

```diff
@@ -35,26 +37,39 @@
 @AllArgsConstructor
 public class LegacyPbaController {
 
-
     private LegacyPbaAccountServiceImpl legacyPbaAccountService;
     private ProfessionalUserServiceImpl professionalUserService;
 
     @ApiOperation(
-            value = "Retrieve pba numbers by user email address"
+            value = "Retrieve Payment Accounts by a User's Email Address"
+    )
+    @ApiParam(
+            name = "email",
+            type = "string",
+            value = "The Email of the User who's Payment Accounts are to be retrieved",
+            required = false
     )
     @ApiResponses({
             @ApiResponse(
                     code = 200,
-                    message = "Details of one or more payment accounts",
+                    message = "Details of one or more Payment Accounts",

```

Comment:
all letters after first should be small caps

Reply:
Payment Accounts is a name, similar to how Organisation is, thus names are spelled with a capital letter

Reply:
ok fine